src/modules/llm/use-cases/interpret-viability-for-crop-sowing/promp-template.ts

```diff
@@ -0,0 +1,67 @@
+import { PromptTemplate } from "@langchain/core/prompts";
+
+// Prompt Template
+export const viabilityAnalysisPrompt = new PromptTemplate({
+  inputVariables: ["crop", "language", "stagesInfo"],
+  template: `
+  You are an expert agronomist.
+  
+  Analyze the following data to determine if it's a good idea to plant {crop} at the specified time and location. Provide your recommendation in {language}.
+  
+  Your response should start with "Hi, Thanks for using Cropilot" in {language}.
+  
+  **Output Format (Please follow this structure exactly):**
+  
+  - Output format: Markdown Language .md
+  
+  1. **Introduction**
+  
+     - Begin with the greeting: "Hi! Thanks for using Cropilot" in {language}.
+     - Simple executive resume of the analysis resume of all the data targeted for someone who does not want to read much
+  
+  2. **Stage-by-Stage Analysis**
+  
+     For each stage, provide:
+  
+     - **Stage Name**
+       - **Duration:** From month X to Y
+       - **Comparative Table:**
+  
+         | Parameter                      | Optimal Value | Actual Average (since 2000) |
+         |--------------------------------|---------------|------------------------------|
+         | Temperature at 2m (°C)         | ...           | ...                          |
+         | Surface Temperature (°C)       | ...           | ...                          |
+         | Precipitation (mm/day)         | ...           | ...                          |
+         | Specific Humidity at 2m (g/kg) | ...           | ...                          |
+         | Wind Speed at 2m (m/s)         | ...           | ...                          |
+         | Surface Soil Moisture          | ...           | ...                          |
+         | Root Zone Soil Moisture        | ...           | ...                          |
+         | Profile Soil Moisture          | ...           | ...                          |
```

Comment:
Tables are not supported by telegram's formatting. We should probably generate charts at this point

https://core.telegram.org/api/entities

Reply:
Lists are also unsupported